mod/item.php

```diff
@@ -449,7 +449,7 @@ function item_post(&$a) {
 			$objecttype = ACTIVITY_OBJ_IMAGE;
 
 			foreach($images as $image) {
-				if(! stristr($image,$a->get_baseurl() . '/photo/'))
+				if(! stristr($image,App::get_baseurl() . '/photo/'))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊